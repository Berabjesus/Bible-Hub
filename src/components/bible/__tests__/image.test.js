import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Image from '../image';

describe('Image bible component', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Image image="www.testimage.com" title="test title" />
      </BrowserRouter>,
    );
    expect(component).toMatchSnapshot();
  });

  it('Renders the Image correctly', () => {
    render(
      <BrowserRouter>
        <Image image="www.testimage.com" title="test title" />
      </BrowserRouter>,
    );

    expect(screen.getByAltText('test title'));
  });
});
