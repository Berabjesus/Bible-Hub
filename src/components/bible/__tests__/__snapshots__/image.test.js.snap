// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image bible component -  matches the snapshot 1`] = `
Array [
  <div
    className="position-absolute w-100 h-100 background "
    style={
      Object {
        "background": "url(www.testimage.com) no-repeat center/cover",
      }
    }
  />,
  <div
    className="w-100 h-100 position-absolute"
  >
    <img
      alt="test title"
      className="w-100 h-100 img"
      src="www.testimage.com"
    />
  </div>,
]
`;
